src/server.c

```diff
@@ -740,6 +796,22 @@ dictType clientDictType = {
     .keys_are_odd = 0           /* a client pointer is not an odd pointer */            
 };
 
+kvstoreType kvstoreBaseType = {
+    NULL, /* kvstore metadata size */
+    NULL, /* dict metadata size */
+    NULL, /* can free dict */
+    NULL, /* on kvstore empty */
+    NULL, /* on dict empty */
+};
+
+kvstoreType kvstoreExType = {
+    kvstoreMetadataBytes,       /* kvstore metadata size */
+    kvstoreDictMetadataBytes,   /* dict metadata size */
+    kvstoreCanFreeDictIfNeeded, /* can free dict */
+    kvstoreOnEmpty,             /* on kvstore empty */
+    kvstoreOnDictEmpty,         /* on dict empty */
+};
```

Comment:
```suggestion
kvstoreType kvstoreExType = {
    kvstoreMetaBytes,       /* kvstore metadata size */
    kvstoreDictMetaBytes,   /* dict metadata size */
    kvstoreCanFreeDict,     /* can free dict */
    kvstoreOnEmpty,         /* on kvstore empty */
    kvstoreOnDictEmpty,     /* on dict empty */
};
```
Try to keep names not too long

Reply:
Done.